.vscode/settings.json

```diff
@@ -22,5 +22,6 @@
         "**/*.pyc": true,
         "**/*.pyo": true,
         "**/*.dist-info": true
-    }
+    },
+    "python.pythonPath": "${workspaceRoot}/venv/bin/python"
```

Comment:
Why are you checking this in?  Can we not checking user specific path settings here?

Reply:
agreed.. also $ make install  will populate ${workspaceRoot}/bin/python.. so this seems pretty user specific

Reply:
My fault; I'll back it out.